src/controllers/users.controller.ts

```diff
@@ -0,0 +1,71 @@
+import {ValidatedRequest} from 'express-joi-validation';
+import * as userValidation from '../validation/validation-schemas';
+import UserService from '../services/user.service';
+import {Request, Response} from 'express';
+import {RESPONSE_MESSAGES} from '../config/messages';
+
+export class UsersController {
```

Comment:
why do you need class if every method is static

Reply:
fixed for service and controller